script/constants/buckets.js

```diff
@@ -0,0 +1,21 @@
+const {
+  DEVELOPMENT,
+  PREVIEW,
+  PRODUCTION,
+  STAGING,
+  VAGOVDEV,
+  VAGOVSTAGING,
+} = require('./environments');
+
+const hostnames = require('./hostnames');
+
+const bucket = 'https://s3-us-gov-west-1.amazonaws.com';
+
+module.exports = {
+  [DEVELOPMENT]: `${bucket}/${hostnames[DEVELOPMENT]}`,
+  [PREVIEW]: `${bucket}/${hostnames[PREVIEW]}`,
+  [PRODUCTION]: `${bucket}/${hostnames[PRODUCTION]}`,
+  [STAGING]: `${bucket}/${hostnames[STAGING]}`,
+  [VAGOVDEV]: 'https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com',
+  [VAGOVSTAGING]: `${bucket}/${hostnames[VAGOVSTAGING]}`,
+};
```

Comment:
Stolen from here, https://github.com/department-of-veterans-affairs/vets-website/pull/8855.

This will be moved into a shared directory, so that Webpack can access it for its source maps config.